examples/tof-viewer/src/ADIView.cpp

```diff
@@ -289,7 +289,12 @@ void ADIView::_displayAbImage() {
 
         auto camera = m_ctrl->m_cameras[static_cast<unsigned int>(
             m_ctrl->getCameraInUse())];
-        camera->normalizeABdata(m_capturedFrame.get(), true);
+
+        if (getLogImage() || (getLogImage() && getAutoScale())) {
+            camera->normalizeABdata(m_capturedFrame.get(), true);
+        } else if (getAutoScale()) {
+            camera->normalizeABdata(m_capturedFrame.get(), false);
+        }
```

Comment:
Can you simplify this a little bit? On both if-branches you are asking about getAutoScale() and want it to be true, so why not do some thing like:
```
if (getAutoscale()) {
    camera->normalizeABdata(m_capturedFrame.get(), getLogImage());
}
```
Also the logic in the 1st if branch doesn't make sense:
you are asking if (condition1 OR (condition1 AND condition2)). If you are in the situation where condition1=TRUE, it doesn't matter the state of condition2.